packages/cmf-cqrs/src/middleware/socketMiddleware.test.js

```diff
@@ -27,6 +27,20 @@ const state = {
 	},
 };
 
+describe('hasWebSocketProtocol', () => {
+	it('should return true for Web Socket url', () => {
+		expect(isWebSocketAbsoluteUrl('ws://provider/api/v1/ws/relay')).toBeTruthy();
```

Comment:
i think you should test against `toBe(true)`, truthy can be many thing else than a true boolean